src/WebDav.Client/Request/SearchParameters.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Xml.Linq;
+
+namespace WebDav.Client.Request
+{
+    public class SearchParameters
```

Comment:
I'd recommend initializing collections like with empty lists (similar to other `XxxParameters` classes).

Reply:
nit: also the formatting of the class is a bit off.

Reply:
Fixed